src/v/coproc/gen.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This commit just replaces the same exact gen.json file that was removed in an earlier commit